test/JDBC/expected/BABEL-3317.out

```diff
@@ -0,0 +1,44 @@
+SET PARSEONLY ON;
+go
+CREATE TABLE t1(a int);
+INSERT INTO t1 values (1);
+SET PARSEONLY OFF;
+SELECT * FROM t1;
+go
+~~ROW COUNT: 1~~
+
+~~START~~
+int
+1
+~~END~~
+
+
+
+SET PARSEONLY ON;
+GO
+INSERT INTO t1 values (2);
+GO
+SET PARSEONLY OFF;
+GO
+SELECT * FROM t1
+GO
+~~START~~
+int
+1
+~~END~~
+
+
+SET PARSEONLY ON;
```

Comment:
What are you trying to test here? It would be nice to have some explanations, especially for corner cases.